README.md

```diff
@@ -1,12 +1,15 @@
 # react-native-launch-arguments
 
-React Native module to get launch arguments.
+React Native module to get launch arguments. Make passing parameters from testing tool to react native super easy. 
 
-Mostly it's made for using [`launchArgs parameter of device.launchApp method`](https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#7-additional-launch-arguments) of [Detox](https://github.com/wix/Detox/)
+Mostly it's made for using 
+* [`launchArgs parameter of device.launchApp method`](https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#7-additional-launch-arguments) of [Detox](https://github.com/wix/Detox/)
+* [`optionalIntentArguments (Android)` and `processArguments (iOS)`](http://appium.io/docs/en/writing-running-appium/caps/) parameters with [Appium](http://appium.io/)
 
 **iOS**: it takes data from `[[NSProcessInfo processInfo] arguments]`
 
-**Android**: it takes data from `currentActivity.getIntent().getBundleExtra("launchArgs")`
+**Android**: it takes data from [`currentActivity.getIntent().getBundleExtra("launchArgs")`](https://developer.android.com/reference/android/content/Intent#getBundleExtra(java.lang.String)) together with [`currentActivity.getIntent().getStringExtra("launchArgs")`](https://developer.android.com/reference/android/content/Intent#getStringExtra(java.lang.String))
+* see [`ADB docs`](https://developer.android.com/studio/command-line/adb#IntentSpec) for details
```

Comment:
can you please leave the same code style (here and in .java file) as it was before, thanks

Reply:
ok. My codestyle was different. I reverted it .